schedule/functional/extra_tests_in_textmode_mod_desktop.yaml

```diff
@@ -0,0 +1,7 @@
+name:           extra_tests_in_textmode_mod_desktop
```

Comment:
```suggestion
name:           extra_tests_textmode_mod_desktop
```

Reply:
The other yaml files are called `extra_tests_in_textmode_…` too, so that wouldn't be consistent then.

Reply:
Yes, because I created them before we agreed on the new names.
They should also be changed.

The filename of the new YAML should be extra_tests_textmode_mod_desktop.yaml